src/vs/editor/common/config/editorOptions.ts

```diff
@@ -4929,6 +4935,13 @@ export const EditorOptions = {
 	ariaLabel: register(new EditorStringOption(
 		EditorOption.ariaLabel, 'ariaLabel', nls.localize('editorViewAccessibleLabel', "Editor content")
 	)),
+	ariaAssertiveInlineSuggestion: register(new EditorBooleanOption(
+		EditorOption.ariaAssertiveInlineSuggestion, 'ariaAssertiveInlineSuggestion', false,
```

Comment:
aria assertive sounds like an implementation detail. Can we have a more user friendly name?